common/persistence/historyStore.go

```diff
@@ -456,13 +456,13 @@ func (m *historyV2ManagerImpl) readHistoryBranch(
 			// In that case we don't validate history continuousness for the first page
 			// TODO: in this case, some events returned can be invalid(stale). application layer need to make sure it won't make any problems to XDC
 			if defaultLastEventID == 0 || token.LastEventID != defaultLastEventID {
-				logger.Error("Corrupted non-continuous event batch",
+				logger.Error("Corrupted non-contiguous event batch",
```

Comment:
This is a good rule: "log error or return it but not both". I know there is no such thing as "too many logs" but still.

Reply:
i agree, but this situation is different, this layer has enough info about tree id, branch id, event id.
new logs try to add more details about workflow / run id